RecoTracker/DeDx/test/ProbQXYFromMiniAOD_cfg.py

```diff
@@ -0,0 +1,42 @@
+import FWCore.ParameterSet.Config as cms
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+from Configuration.Eras.Era_Run2_2018_cff import Run2_2018
+process = cms.Process('RECO2',Run2_2018)
+
+options = VarParsing.VarParsing('analysis')
+options.register('globalTag',
+                 "auto:run2_mc", # default value
```

Comment:
just noting *en passant* that the default choice of `auto:run2_mc` (i.e. 2015+2016 run2 simulation) does not match well with the default era (`Run2_2018`) even if the GT is overridden in the actual test.